WebRoot/js/zimbraMail/prefs/controller/ZmPrefController.js

```diff
@@ -328,6 +328,12 @@ function () {
 		}
 	}
 	this._lefttoolbar.getButton(ZmOperation.SAVE).setToolTipContent(ZmMsg.savePrefs);
+
+	var children = this._lefttoolbar.getHtmlElement().children;
+	if(children && children[0]) {
+		children[0].setAttribute('role', 'region');
+		children[0].setAttribute('aria-label', ZmMsg.leftToolbar);
```

Comment:
can we use a variable for `children[0]`?

Reply:
Updated.